### DISCUSSION TITLE
Is it possible to call methods from a model for list operations?

### DISCUSSION THREAD

**Discussion by NuktukDev** [Laravel-Backpack/community-forum#968] [Q&A (Help)]
Put simply, I use [Laravel-nested-sets](https://github.com/lazychaser/laravel-nestedset) and want to use the ->toFlat() method on the model, and use the outcome for my listing data.

How would I go about doing this?

**Comment by jcastroa87** [comment]
Hello @NuktukDev 

If you want to customize the operation you can follow our [docs](https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries#the-operation-logic-1) Instructions.

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity I will close the issue, but please feel free to re-open or create a new one if needed.

Cheers.
